pkgs/desktops/gnome/extensions/extensionOverridesPatches/tophat_at_fflewddur.github.io.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/extension.js b/extension.js
+index 60396f8..ec0731f 100644
+--- a/extension.js
++++ b/extension.js
+@@ -32,7 +32,7 @@ let Mem = null;
+ let Net = null;
+ try {
+     // eslint-disable-next-line no-unused-vars
+-    GTop = imports.gi.GTop;
++    GTop = imports.gi.GIRepository.Repository.prepend_search_path('@gtop_path@');
```

Comment:
This will not work. `prepend_search_path` returns nothing. you need to preserve `GTop = imports.gi.GTop`.

Reply:
Thx for the input. I've update it.